software/seafile.yml

```
@@ -10,6 +10,9 @@ platforms:
   - C
 tags:
   - File Transfer & Synchronization
+  - Knowledge Management Tools
+  - Office Suites
+  - Wikis
```

```suggestion
  - Groupware
```

This is the only acceptable change, similar to what we already do with other collaborative suites.